packages/core/.storybook/mayflowerTheme.js

```diff
@@ -3,17 +3,17 @@ import logo from './mayflower-logo.png';
 export default create({
   base: 'light',
 
-  // colorPrimary: 'hotpink',
+	colorPrimary: '#388557',
   colorSecondary: '#14558F',
 
   // UI
   appBg: '#F2F2F2',
-  // appContentBg: 'silver',
+	appContentBg: '#FFFFFF',
   appBorderColor: '#DCDCDC',
-  appBorderRadius: 6,
+  appBorderRadius: 0,
 
   // Typography
-  fontBase: '"Noto Sans VF", "Noto Sans", "Helvetica", "Arial", sans-serif',
+  fontBase: '"Noto Sans VF", "Noto Sans", sans-serif',
```

Comment:
can we put back the two fallback fonts, to be consistent with mayflower font settings?

Reply:
Yes, but not those two, they don't match stylistically. I will follow up with some that do.

Reply:
Probably should patch the main repo too.

Reply:
I think the purpose here is to control the fallback fonts, when user turns off webfonts in their browsers. They'll need to be popular system fonts